src/Provider/OpenRouteService/README.md

```diff
@@ -0,0 +1,27 @@
+# OpenRouteService Geocoder provider
+[![Build Status](https://travis-ci.org/geocoder-php/open-route-service-provider.svg?branch=master)](http://travis-ci.org/geocoder-php/open-route-service-provider)
```

Comment:
I would put openrouteservice (without dashes) since it is how they refer to it (see https://github.com/GIScience/openrouteservice)